scripts/main.js

```diff
@@ -52,11 +52,11 @@ module.exports = (robot) => {
         }
     })
     robot.error((err, res) => {
-        console.log(err)
-        robot.logger.error("DOES NOT COMPUTE")
+        console.log(err.message)
+        robot.logger.error(err.message)
```

Comment:
@Mikanebu ok let's keep it, but have you tested it? Does it actually have `error.message` property? previously it was just a string